tools/migrate_chart/Readme.md

```diff
@@ -0,0 +1,17 @@
+# Chart Migrating Tool
+
+Harbor supports two different ways to storage the chart data.
+
+   1. stored in Harbor registry storage directly via OCI API.
+   2. stored in Harbor hosted chartmuseum backend via chartmuseam's API
+
+This tool used to migrating the helm charts stored in the chartmuseum backend to Harbor OCI registry backend.
+
+## Usages
+
+Run command below:
+
+```
+docker run -it --rm -v {{your_chart_data_location}}:/chartmuseum/ goharbor/prepare:{{version}}
```

Comment:
wrong image name.
And how does user specify the Harbor endpoint and username/password?